locations/dict_parser.py

```diff
@@ -159,6 +159,7 @@ class DictParser:
         "store-zip",
         "store-zip-code",
         "store-zipcode",
+        "zipCd",
```

Comment:
How common is this? Would it be better to just do it in the spider?

Reply:
Agreed, I've removed it from DictParser as I can't see if ever appearing elsewhere.